GOVERNANCE.md

```diff
@@ -65,6 +65,48 @@ meeting. In both cases the decision must be seconded by a majority of the Team
 or else the discussion will continue. Simple majority wins. Only Active
 Members participate in a vote.
 
+
+## Electrion process for Non-Impact Voting CPC members
+
+As per the CPC charter, the non-impact projects elect 2 voting CPC members:
+
+> up to two Voting members may be nominated by the non Impact projects
+  based on a process set by the CPC.
+
+These members will be elected as follows:
+
+* Once a year an issue will be opened in the CPC announcing that the period for
+  nominations is open.
+
+* Any member from one of the non-impact projects can self-nominate by posting a
+  reply which includes the following:
+
+  * Projects they are part of
+  * Description of their background
+  * The reasons they would like to act as the non-impact CPC voting member
+  * Confirmation that they can devote the required time/effort.
+  * Their employer  (if applicable).  This is needed due to the limitations
+    on maximum employer representation in the CPC voting members.
+
+* The nomination period will be open for 2 weeks at which point the issue will be locked.
+
+* Once the nomination period closes, the projects will be asked to confirm membership
+  as identified by the nominees.
+
+* Each project will nominate two `voters` through a process of their choosing.
+
+* Each voter will be able to vote for as many or as few candidates as they wish.
+
+* Those candidates who receive the greatest number of votes will be confirmed as
```

Comment:
should we maybe do a run off election rather than coin toss?

Reply:
Definitely.

Reply:
How about using the number of projects they are actively part of as a selector rather than a coin toss?

Reply:
We had discussed having multiple rounds but there was some concern over asking people to manage and participate in a number of voting rounds.  @sendilkumarn I'm not sure the number of projects is necessarily a good selector as it gets very subjection to assess the level of participation in a project.

Reply:
I would prefer holding runoff elections in the event of a tie as I'm not too keen on using random selection to choose who represents a potentially large group of projects on the CPC. That said, if the consensus is to avoid multiple rounds of voting, I don't have a better suggestion other than one of the more complex voting systems of which I am also not a big fan.

Reply:
If the top is a two-way tie, those two should be the two voting members. That makes sense right?
If the second position is a tie, then random, but I suggest giving the candidates the chance to concede to each other first?
If the top position is a three-way tie or more, then maybe we should have a second election?

Reply:
I like that @Jonahss as it should avoid a run off in most cases and only requires a coin toss when necessary rather than defaulting to that.

Reply:
@Jonahss  if its a 2 way tie I had assumed it would be those 2 based on the existing wording.  Do you think I should clarify that further?

Writing in a chance for consensus first makes sense to me, I'll add that.

I'm neutral on re-vote or random, but you might still end up with three-way or more tie on a revote as well and we'd have what to decide to do in that case.

Reply:
I assumed that the revote would be limited to just the tied candidates. Is that what you were thinking too?

2 way tie case doesn't need to be clarified I guess.

Reply:
When I looked at the text again I added some extra clarification, @Jonahss do you think that's better?